build/zip.py

```diff
@@ -5,9 +5,11 @@
 import zipfile
 
 LINUX_BINARIES_TO_STRIP = [
+  'chromedriver',
   'electron',
   'libffmpeg.so',
-  'libnode.so'
+  'libnode.so',
+  'mksnapshot'
```

Comment:
these binaries shouldn't be stripped as part of the zip process, i don't think. i missed the original review that added this though :(

Reply:
yes, i agree that these binaries should be zipped, i just don't think that `zip.py` is the right place to do it. I'll reserve further comment until i have a concrete suggestion though :)

Reply:
oops, s/zipped/stripped in last comment